docs/guide/SettlementBC.md

```diff
@@ -45,13 +53,24 @@ To initiate the settlement process, the hub operator selects :
 The position ledgers of the net credit participants are adjusted during Settlement Initiation.
 **Note:** It is important to create the batch settlement object in the way that the settlement is to be completed and finalized.
 1. A **Settlement initiation report**  is generated and used to communicated to the settlement bank the requirements of the settlement.
-1. **Settlement Finalization & settlement account rebalancing**
+1. **Settlement Finalization & settlement account re-balancing**
 This process needs to occur after the settlement bank has applied the settlement changes. In this step the:
    - settlement process is completed and a settlement finalization report has been received from the settlement bank.
    - the net debit participants in the settlement have their position ledgers adjusted.
    - the settlement ledgers are adjusted for all participants to match the transferred amount for the settlement.
    - the settlement ledgers are checked against the real settlement account balances and adjustments processed to ensure that they are aligned.
 
+### Re-balancing function - is not best practice
+It is worth noting that the re-balancing function that is defined in the above settlement finalization process is not the preferred or best-practice approach. 
+This approach was chosen because of regularity requirements and limitations of mechanisms available to implement settlement between participants. I.e. it was designed to work on existing in-place financial solutions. Re-balancing has quite a few drawbacks, and is not considered best practice and should be avoided if possible. 
```

Comment:
```suggestion
This approach was chosen because of regulatory requirements and limitations of mechanisms available to implement settlement between participants. I.e. it was designed to work on existing in-place financial solutions. Re-balancing has quite a few drawbacks, and is not considered best practice and should be avoided if possible. 
```